tests/test_basic.py

```diff
@@ -185,6 +201,24 @@ def runTest(self):
         ])
 
 
+class RandomSeedClassTestNP(RandomlyPluginTester, TestCase):
```

Comment:
I don't think we need this test case - we have a test case that checks that random seeds in general are fixed during classes, and since it's part of the contract of `nose-randomly` that the various random states are reset at the same time, we shouldn't need to test again that the `numpy`'s specific random state is reset during classes.